src/NovelRenderingService.cpp

```diff
@@ -52,12 +70,14 @@ bool NovelRenderingService::initializeRenderPipeline(const int displayNumber) {
   }
   _openGLContext = SDL_GL_CreateContext(_window.get());
   SDL_GL_MakeCurrent(_window.get(), _openGLContext);
-  SDL_GL_SetSwapInterval(0);
 
   if (!gladLoadGL()) {
     std::cerr << "ERROR: Failed to initialise glad." << std::endl;
     return -1;
   }
+    // During init, enable debug output
+    glEnable              ( GL_DEBUG_OUTPUT );
+    glDebugMessageCallback( MessageCallback, 0 );
```

Comment:
I believe this was temporarily disabled since it requires 4.3 or higher and the minimum is currently 3.2.

Windows (and some Linux configurations) will actually give you explicitly the version requested, in which case:
* The versioning logic needs to request 3.2 as a minimum and allow a later version if availble
* The `glDebugMessageCallback` needs to be "light up" based on what version is actually being used

It might be easiest to log a bug for that and to set 4.3 as the explicit version right now. @RubyNova ?

Reply:
Yeah that might be wise. I wanted to deal with it later but if its being reenabled we should put it in the `NovelDebugService` as discussed and leave a TODO to revisit the issue later. :)

Reply:
I can drop it for now and leave it for you folks to reimplement at a later time :)